e2e_correctness/query_neo_mem.py

```diff
@@ -0,0 +1,275 @@
+"""
+This module queries Memgraph and Neo4j and creates Graph from JSON exported from Memgraph and 
+JSON from APOC from Neo4j
+
+As of 17.7.2023. when importing data via Cypherl, new ids is given to each node in Memgraph and Neo4j.
+
+When exporting data Memgraph export_util uses internal Memgraph ids to export data. 
+
+To overcome issue of different ids between Neo4j and Memgraph, we use ids on properties.
```

Comment:
How does this function?

Reply:
Consider rephrasing the docstring like this:
```suggestion
To overcome the issue of different internal IDs in Neo4j and Memgraph, we use the `id` node property as identifier.
```